package.json

```diff
@@ -18,7 +18,8 @@
     "mongoose": "^5.1.5",
     "morgan": "~1.9.0",
     "short-id": "0.1.0-1",
-    "shortid": "^2.2.8"
+    "shortid": "^2.2.8",
```

Comment:
I'm assuming you are using this one. Can you remove `short-id` above it. `yarn remove short-id`.